### PR TITLE
Add Siriusnet chain and icon

### PR DESCRIPTION
**Pull request by warewar1**

### CODE REVIEW

File: _data/chains/eip155-67390.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://macaucasinolisboa.xyz",
+  "shortName": "mcl",
+  "chainId": 67390,
+  "networkId": 67390,
+
+  "explorers": [{
+    "name": "siriusnetscan",
+    "url": "https://siriusnetscan.io",
```
Comment (ligi) [inline _data/chains/eip155-67390.json:19]: I cannot reach this URL

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-67390.json (hunk 1 above)
Comment (warewar1) [reply _data/chains/eip155-67390.json]: Solved

**Review by warewar1 — COMMENTED** [review]
solved

**Comment by warewar1** [comment]
Solved

**Comment by warewar1** [comment]
Please aprove!

**Review by ligi — CHANGES_REQUESTED** [review]
This is not a good change:

image

**Pull request closed without merging**